src/libraries/iCubDev/CMakeLists.txt

```diff
@@ -4,10 +4,17 @@
 
 project(iCubDev)
 
+# Add IDL files generation from thrift
+set(idl_files include/iCub/rawValuesPublisher.thrift)
+set(doc_files include/iCub/rawValuesPublisher.xml)
+# yarp_add_idl(IDL_GEN_FILES ${idl_files})
+yarp_idl_to_dir(INPUT_FILES ${idl_files} OUTPUT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include/iCub PLACEMENT MERGED)
```

Comment:
I think that the Thrift generation should be somehow put under a switch guard as we agreed upon, while awaiting `yarp-3.10.0`.

Is this guaranteed?

Reply:
No thrift generation is fine. The only problem with 3.10 is related to the ParamParserGenerator plugin, that's why I kept those files of the plugins modified and with comments and disabled their generation by default. No problem instead with the IDL_GENERATION